apps/dashboard/src/components/contract-functions/contract-function.tsx

```diff
@@ -408,12 +422,22 @@ export const ContractFunctionsPanel: React.FC<ContractFunctionsPanelProps> = ({
               </TabList>
               <TabPanels h="auto" overflow="auto">
                 {writeFunctions.length > 0 && (
-                  <TabPanel>
+                  <TabPanel p="0">
+                    <Input
```

Comment:
@MananTank The input is transparent, making the content to be overlapped. Wdyt is the way to fix it?

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/VtULqGKSp33n0VHOzNJK/5e37bf9f-687a-4a2c-b70f-d82f3262591d.png)

Reply:
oh wait, it's just z-index LOL